src/main/java/gregtech/common/metatileentities/multi/multiblockpart/appeng/MetaTileEntityMEInputBus.java

```diff
@@ -190,13 +267,16 @@ public NBTTagCompound writeToNBT(NBTTagCompound data) {
         data.setBoolean(WORKING_TAG, this.workingEnabled);
         NBTTagList slots = new NBTTagList();
         for (int i = 0; i < CONFIG_SIZE; i++) {
-            ExportOnlyAEItem slot = this.aeItemHandler.inventory[i];
+            ExportOnlyAEItemSlot slot = this.getAEItemHandler().getInventory()[i];
```

Comment:
Will this change preserve compat with existing worlds?

Reply:
Yes, its tested to work fine